test/intervals/TestIntervals.jl

```diff
@@ -183,7 +181,9 @@ facts("IntervalCollection") do
 
 
     context("Show") do
-        nullout = open("/dev/null", "w")
+        nullstream = @windows ? "NUL" : "/dev/null"
+        nullout = open(nullstream, "w")
```

Comment:
I can't write anything to DevNull:
```
julia> write(DevNull, "Hi")
ERROR: Base.DevNullStream does not support byte I/O
 in write at io.jl:70
 in write at ascii.jl:99
 in eval at ./boot.jl:264

julia> show(DevNull, 15)
ERROR: Base.DevNullStream does not support byte I/O
 in write at io.jl:70
 [inlined code] from ascii.jl:99
 in show at show.jl:143
 in eval at ./boot.jl:264
```
The docs only show it as used in the run command, e.g. `run(`cat test.txt` |> DevNull)`